writer.go

```diff
@@ -70,7 +70,7 @@ func open(paths []string) ([]zapcore.WriteSyncer, func(), error) {
 	for _, path := range paths {
 		sink, err := newSink(path)
 		if err != nil {
-			openErr = multierr.Append(openErr, fmt.Errorf("couldn't open sink %q: %v", path, err))
+			openErr = multierr.Append(openErr, fmt.Errorf("couldn't open sink %q: %w", path, err))
```

Comment:
while we're playing here, consider also changing the error message to conform to [Code Review Comments > Error Strings](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).

```suggestion
			openErr = multierr.Append(openErr, fmt.Errorf("open sink %q: %w", path, err))
```